js/everything.js

```diff
@@ -518,16 +518,20 @@ $(function() {
     });
 
     // Some demo video's audio, feel free to add more
-    var demos = ["koJv-j1usoI", "5cJIvC6AAkc" , "EBerFisqduk", "DlKXJ906pd8"];
-    function playDemo(){
+    var demos = [
+        "koJv-j1usoI", // The Glitch Mob - Starve the Ego, Feed the Soul
+        "5cJIvC6AAkc", // Family Force 5 - Dance Or Die Official Music Video
+        "EBerFisqduk", // Cazzette - Together (Lost Kings Remix)
+        "DlKXJ906pd8" // Ronald Jenkees - Throwing Fire
+    ];
+    function pickDemo() {
```

Comment:
Can we move this function outside this block of code? It should be above `$(function() {`